docs/DeveloperGuide.adoc

```diff
@@ -76,3 +76,7 @@ OSes. Convert the actual.txt to the format used by your OS using some https://kb
 * Problem: Test fails during the very first time. +
 Solution: The output of the very first test run could be slightly different because the program
 creates a new storage file. Tests should pass from the 2nd run onwards.
+
+=== Adding more sort functions
+Refer to `UniquePersonList class` and `Person class`. +
```

Comment:
should be `UniquePersonList` class, `Person` class, instead of `UniquePersonList class` and `Person class`.

Actually what is this?

Reply:
This is just to point the developer in the right direction in the event he has to edit or add more comparators.